config/converters.py

```diff
@@ -0,0 +1,8 @@
+class SchemaFileConverter:
+    regex = r"(?:xml|caselaw|akn-modified)\.xsd"
```

Comment:
Could we add `https://raw.githubusercontent.com/nationalarchives/ds-caselaw-marklogic/main/src/main/ml-schemas/` as a constant here? (Something like `SCHEMA_ROOT`) then return the full URL in `to_url`?

Reply:
Rewritten the failing tests and added SCHEMA_ROOT. They failed because I turned caching on...